src/components/StudentTable.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Button,
+  Grid,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@mui/material";
+import MoreVertIcon from "@mui/icons-material/MoreVert";
+import Paper from "@mui/material/Paper";
+import React, { useEffect } from "react";
+import {
+  IDispatchToProps,
+  IStudentRaw,
+  IStudentState,
+} from "../state/ducks/student/types";
+import moment from "moment";
+type AllProps = IDispatchToProps & IStudentState;
+function StudentTable({ fetchStudents, data }: AllProps) {
+  useEffect(() => {
+    fetchStudents();
+  }, [fetchStudents]);
+
+  const specificDate = (time: string) =>
+    moment(time).format("MMM D, YYYY [at] hh.mm A");
```

Comment:
create a file helper.ts
move this function to that file and import it here.

Reason this funciton is utility function not related to react it self and can be called from another components.